src/main/scala/io/github/ilyazinovich/dmmf/DomainTypes.scala

```diff
@@ -66,8 +61,6 @@ object ZipCode {
 
 object OrderId {
 
-  private def apply(string: String): OrderId = new OrderId(string)
```

Comment:
Please, leave private constructors. They make it impossible to construct invalid instances of case classes.

Reply:
This code is not used.
Your constructor is still private. You made it private in case class declaration.

Reply:
But I can instantiate `OrderId` using constructor in test.

Reply:
Ok